includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -1196,38 +1398,62 @@ public static function filter_terms_clauses_for_description_search( $clauses, $t
 	 * Show notices for changes to amp_validation_error terms.
 	 */
 	public static function add_admin_notices() {
-		if ( ! ( self::TAXONOMY_SLUG === get_current_screen()->taxonomy || AMP_Invalid_URL_Post_Type::POST_TYPE_SLUG === get_current_screen()->post_type ) || empty( $_GET['amp_actioned'] ) || empty( $_GET['amp_actioned_count'] ) ) { // WPCS: CSRF ok.
+		if ( ! ( self::TAXONOMY_SLUG === get_current_screen()->taxonomy || AMP_Invalid_URL_Post_Type::POST_TYPE_SLUG === get_current_screen()->post_type ) ) { // WPCS: CSRF ok.
 			return;
 		}
-		$actioned = sanitize_key( $_GET['amp_actioned'] ); // WPCS: CSRF ok.
-		$count    = intval( $_GET['amp_actioned_count'] ); // WPCS: CSRF ok.
-		$message  = null;
-		if ( self::VALIDATION_ERROR_ACCEPT_ACTION === $actioned ) {
-			$message = sprintf(
-				/* translators: %s is number of errors accepted */
-				_n(
-					'Accepted %s error. It will no longer block related URLs from being served as AMP.',
-					'Accepted %s errors. They will no longer block related URLs from being served as AMP.',
-					number_format_i18n( $count ),
-					'amp'
-				),
-				$count
-			);
-		} elseif ( self::VALIDATION_ERROR_REJECT_ACTION === $actioned ) {
-			$message = sprintf(
-				/* translators: %s is number of errors rejected */
-				_n(
-					'Rejected %s error. It will continue to block related URLs from being served as AMP.',
-					'Rejected %s errors. They will continue to block related URLs from being served as AMP.',
-					number_format_i18n( $count ),
-					'amp'
-				),
-				$count
-			);
+
+		// Show success messages for accepting/rejecting validation errors.
+		if ( ! empty( $_GET['amp_actioned'] ) && ! empty( $_GET['amp_actioned_count'] ) ) {
+			$actioned = sanitize_key( $_GET['amp_actioned'] ); // WPCS: CSRF ok.
+			$count    = intval( $_GET['amp_actioned_count'] ); // WPCS: CSRF ok.
+			$message  = null;
+			if ( self::VALIDATION_ERROR_ACCEPT_ACTION === $actioned ) {
+				$message = sprintf(
+					/* translators: %s is number of errors accepted */
+					_n(
+						'Accepted %s error. It will no longer block related URLs from being served as AMP.',
+						'Accepted %s errors. They will no longer block related URLs from being served as AMP.',
+						number_format_i18n( $count ),
+						'amp'
+					),
+					$count
+				);
+			} elseif ( self::VALIDATION_ERROR_REJECT_ACTION === $actioned ) {
+				$message = sprintf(
+					/* translators: %s is number of errors rejected */
+					_n(
+						'Rejected %s error. It will continue to block related URLs from being served as AMP.',
+						'Rejected %s errors. They will continue to block related URLs from being served as AMP.',
```

Comment:
This looks good:

<img width="926" alt="notice-reject-bulk-action" src="https://user-images.githubusercontent.com/4063887/45891374-c6f00780-bd9b-11e8-9057-e1a1561b3225.png">